src/rust-2018/platform-and-target-support/musl-support-for-fully-static-binaries.md

```diff
@@ -2,6 +2,9 @@
 
 ![Minimum Rust version: 1.1](https://img.shields.io/badge/Minimum%20Rust%20Version-1.1-brightgreen.svg)
 
+(Note: current Rust (as of 1.48) supports static linking with either glibc or
+MUSL, so you can select whichever libc you need or prefer.)
```

Comment:
Are you sure it is 1.48?  I thought static support is in 1.49 (https://github.com/rust-lang/rust/pull/77386).

I would suggest being a little clearer here, and providing an example:

```suggestion
> Note: Starting with Rust 1.49, the Linux glibc targets also support static
> linking the C runtime. This can be done with the [`+crt-static` target feature][crt-static].
> For example, with Cargo:
>
> ```console
> RUSTFLAGS="-C target-feature=+crt-static" cargo build --target x86_64-unknown-linux-gnu
> ```

[crt-static]: ../../../reference/linkage.html#static-and-dynamic-c-runtimes
```

I would also put this at the bottom of this section (just before `## Installing MUSL support`).